<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAircraftComponent</class>
 <widget class="QTabWidget" name="CAircraftComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airports in range</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tb_AircraftInRange">
   <attribute name="title">
    <string>Aircraft in range</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_AircraftsInRange">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="swift::gui::views::CSimulatedAircraftView" name="tvp_AircraftInRange">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_AirportsInRange">
   <attribute name="title">
    <string>Airports in range</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_AirportsInRange">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="swift::gui::views::CAirportView" name="tvp_AirportsInRange">
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::views::CSimulatedAircraftView</class>
   <extends>QTableView</extends>
   <header>gui/views/simulatedaircraftview.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::views::CAirportView</class>
   <extends>QTableView</extends>
   <header>gui/views/airportview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
